check_nightly_success/check-nightly-success/check.py

```diff
@@ -52,6 +52,7 @@ def main(
     now = datetime.now(tz=tz)
 
     latest_success = {}
+    oldest_run_valid = {}
```

Comment:
"valid" is too ambiguous naming here. Let's make it clear that validity is determined by recency.
```suggestion
    has_recent_run = {}
```

Reply:
Thank you. I knew this was a flawed name but struggled to come up with a better one.

Reply:
I settled on `workflow_active_for_max_days` because `has_recent_run` would be true if it ran today/yesterday. What we really mean is "this has been active for at least `max_days_without_success` days."

Reply:
I'm good with that. I was thinking something similar but didn't want to propose an even more verbose rename than I already had.